README.md

```diff
@@ -1 +1,32 @@
 # Projekt z baz danych 2 – Karol Wilk, Jakub Gonet
+
+# Opis
+
+Projekt zakłada stworzenie systemu do rejestracji na zajęcia na uczelni.
+Studenci zostają przypisani do odpowiedniej grupy rocznikowej oraz są w stanie przydzielić punkty na poszczególne terminy zajęć.
+
+Po zamknięciu zapisów moderatorzy roku przypisują studentów do grup uwzględniając ich preferencje. Po zakończeniu przydziału do grup studenci mogą zobaczyć wybrany plan.
+
+# Aktorzy
+
+## Student
+
+- Ma dostęp do swoich danych (dane; rocznik; zapisy, do których jest dodany)
+- Może wybrać przedmioty w ramach zapisów
+- Może przydzielić ograniczoną liczbę punktów w ramach zapisu na dany termin
+- Może zobaczyć przydzielony termin
```

Comment:
W enrollu można zobaczyć wszystkie grupy zajęciowe i ich skład. Czy tu też należy wprowadzić tę funkcjonalność?

Reply:
Tutaj może niekoniecznie, bo to, że ludzie mogą widzieć grupy innych niekoniecznie jest pożądane. Ale jestem otwarty na sugestie.

Reply:
Naturalne wydaje się, żeby student miał dostęp do składu jedynie swojej grupy...
Nie wiem jak tam sprawa z rodo itd. wygląda ale skoro jest w enrollu to chyba może być.
Wydaje mi się że taki dobry będzie taki kompromis.
```suggestion
- Może zobaczyć przydzielony termin
- Może zobaczyć skład grup zajęciowych, których jest częścią
```

Reply:
Okej, to brzmi spoko, dorzucam.